test/e2e/mcs_test.go

```diff
@@ -0,0 +1,101 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/coreos/go-semver/semver"
+	ign3 "github.com/coreos/ignition/v2/config/v3_2"
+	ign3types "github.com/coreos/ignition/v2/config/v3_2/types"
+	ign3_3 "github.com/coreos/ignition/v2/config/v3_3"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	"github.com/openshift/machine-config-operator/test/framework"
+	"github.com/openshift/machine-config-operator/test/helpers"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/uuid"
+)
+
+// TestIgn33Kargs ensures that Kernel arguments are available in the Ingition
+// payloads when the client requests a 3.3.0 Spec.
+func TestIgn33Kargs(t *testing.T) {
```

Comment:
Is there anyway to combine this with :
https://github.com/openshift/machine-config-operator/blob/9d1da2759b62e4b7292f9825e7cfc8483167b427/test/e2e/mcd_test.go#L683

I don't think we need 2 ign tests.

But also for this test it's slow bc you are applying to whole worker pool. Rather, use a custom pool with 1 node for a faster rollout like the link above.